yarn.lock

```diff
@@ -1762,7 +1762,7 @@
     "@types/memoizee" "^0.4.2"
     memoizee "^0.4.14"
 
-"@polkadot/api@^0.77.0-beta.4":
+"@polkadot/api@^0.77.0-beta.4", "@polkadot/api@^0.77.1":
```

Comment:
There shouldn't be any mention of `0.77.0-beta.4` inside yarn.lock anymore. Maybe update the resolutions field in package.json too?

Reply:
Ah, dammit....

Reply:
As in: Yes, totally. Should be fixed now